build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
 	id 'java'
 	id 'org.springframework.boot' version '3.0.6'
 	id 'io.spring.dependency-management' version '1.1.0'
-	id "io.github.kobylynskyi.graphql.codegen" version "5.7.2"	
+	id "io.github.kobylynskyi.graphql.codegen" version "5.8.0"
```

Comment:
Does this give us any new "good to know about" features or did just a maintenance release?

Reply:
Ah yeah not really necessary. I just saw in their GitHub that they added a feature to exclude fields from generating in the java class (would have been handy for the courseMemberships), but this is not in this version yet.